game/src/main/scala/io/aigar/model/Team.scala

```diff
@@ -1,15 +1,90 @@
 package io.aigar.model
 
 import slick.driver.H2Driver.api._
+import slick.lifted.TableQuery
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.Duration
+import scala.concurrent.Await
 
-case class Team(id:Int, teamSecret:String, teamName:String, score:Int)
+case class Team(id: Option[Int], teamSecret: String, teamName: String, score: Int)
```

Comment:
Because when you create a new Team you don't know the ID before having added it to the database.